generator/regions/region_base.cpp

```diff
@@ -4,11 +4,26 @@
 
 #include "base/assert.hpp"
 #include "base/control_flow.hpp"
+#include "base/string_utils.hpp"
 
 namespace generator
 {
 namespace regions
 {
+
```

Comment:
we don't put empty lines after the opening namespace brace (and before the closing one neither)

Reply:
fixed

Reply:
fixed